components/shell/fish/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017, Longrin Wischnewski.
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		fish
+COMPONENT_VERSION=	2.6.0
+COMPONENT_FMRI=         shell/fish
+COMPONENT_CLASSIFICATION=System/Shells
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=	https://fishshell.com
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:7ee5bbd671c73e5323778982109241685d58a836e52013e18ee5d9f2e638fdfb
+COMPONENT_ARCHIVE_URL=	https://fishshell.com/files/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=      GPLv2
+COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license 
+COMPONENT_SUMMARY=	Fish is a smart and user-friendly command line shell for macOS, Linux, and the rest of the family.
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+CONFIGURE_BINDIR.32 = $(USRBINDIR)
+CONFIGURE_BINDIR.64 = $(USRBINDIR)
+
+COMPONENT_PRE_CONFIGURE_ACTION = ($(CLONEY) $(SOURCE_DIR) $(@D))
+
+CONFIGURE_OPTIONS.32 += --sysconfdir=/etc
+CONFIGURE_OPTIONS.64 += --sysconfdir=/etc/$(MACH64)
+
+build: $(BUILD_32_and_64)
```

Comment:
I thought it's a good idea to offer this built for 32 bit systems too as OI seems to have some kind of support for it. But clearly I'm wrong :-)

Fixed